spec/amp-actions-and-events.md

```diff
@@ -22,9 +22,8 @@ The `on` attribute is used to install event handlers on elements. The events tha
 
 The value for the syntax is a simple domain-specific language of the form:
 
-```javascript
-eventName:targetId[.methodName[(arg1=value, arg2=value)]]
-```
+[sourcecode:javascript]
+eventName:targetId[.methodName[(arg1=value, arg2=value)]][/sourcecode]
```

Comment:
nit: Should the `[/sourcecode]` be on a new line?

Reply:
Prettier formats it this way